### DISCUSSION TITLE
Stating /admin_kill instead of /killed

### DISCUSSION THREAD

**Discussion by VanLyden** [c0nD/FluffyBOT#54] [Bug Report]
<img width="1001" alt="Screenshot 2023-02-27 at 3 05 51 AM" src="https://user-images.githubusercontent.com/115385250/221520846-255b7fef-8f0b-4faf-bb8f-04bcb5b87354.png">
I had a new member testing out Fluffy in the sandbox channel and noticed that the error was telling them to call /admin_kill instead of /killed. Dunno if this is because it was a sandbox channel or if it applies to the normal callouts too.

**Comment by c0nD** [comment]
change the error message in `finals.py`
